### ISSUE TITLE
[LANGUAGE] Node 16

### ISSUE THREAD

**Issue by dylburger** [PipedreamHQ/pipedream#2030]
I'd like Pipedream to support the Node 16 runtime.

**Comment by n-h-n** [comment]
@ctrlaltdylan how far out is this looking?

**Comment by ctrlaltdylan** [comment]
@n-h-n I wish I could give a release date, but none at this time. What kind of use case do you need v16 for?

**Comment by n-h-n** [comment]
Some newer packages like discord's API SDK require node 16+ @ctrlaltdylan

**Comment by cedrixm** [comment]
+1 
discord sdk, discord.js etc. 
blocking not to have this, as given recent changes some features aren't available anymore with the current libraries

**Comment by tonyxiao** [comment]
Node 18 is now currently LTS. So perhaps we should rename the issue to node 18 support?

**Comment by dylburger** [comment]
@tonyxiao Yes, we were discussing this internally as well, and I changed the title to reflect that.

**Comment by Matt2012** [comment]
See also langchain requires 18+. Example of current nodejs baseline expectations.

**Comment by ctrlaltdylan** [comment]
We’re excited to support that Node v18 is now the default Node.js runtime on Pipedream. The next time you deploy an existing workflow, or create a new one, that workflow will use a Node 18 execution environment. [See the docs](https://pipedream.com/docs/code/nodejs/).

You should expect no major changes / issues upgrading to Node 18. Major npm packages have supported Node 18 for some time. Many packages that failed to work on our v14 runtime — such as [discord.js](https://github.com/discordjs) — should now work in Node 18. Node 18 also introduces a native fetch API. Please see [the Node 16](https://nodejs.org/en/blog/release/v16.0.0) and [Node 18 changelogs](https://nodejs.org/en/blog/release/v18.0.0) for more detail.

Node.js continues to be our most popular runtime. You can write any Node.js code in a workflow, use any npm package, programmatically end, delay, or retry a workflow, and more. [Read the Node.js docs](https://pipedream.com/docs/code/nodejs/) to learn more.
